BedrockServer.h

```diff
@@ -91,6 +91,49 @@ class BedrockServer : public SQLiteServer {
     // can do about these, except not shut down. If we're standing down, we could keep them, but they break our
     // checking against whether the main command queue is empty, and fall into the same category of us being unable to
     // distinguish them from commands that may have already started.
+    //
+    // Notes on timing out a shutdown.
+    // Here's how timing out a shutdown works:
+    // 1. When beginShutdown() is called, it sets a one minute timer. It proceeds to shut down the node as normal.
+    // 2. When shutdown progresses far enough that we can shut down the sync node, we set the timeout for the sync node
+    // to whatever portion of our minute is remaining (minus 5 seconds, to allow for final cleanup afterward).
```

Comment:
should we add a note here that if we've used the entire minute (well actually if we've used 55 seconds), the timeout is actually 1s?

Reply:
It's 1us, but sure.